app/controllers/concerns/shopify_app/ensure_authenticated_links.rb

```diff
@@ -28,8 +28,8 @@ def splash_page_with_params(params)
     def redirect_to_splash_page
       redirect_to(splash_page)
     rescue ::ShopifyApp::ShopifyDomainNotFound => error
-      Rails.logger.warn("[ShopifyApp::EnsureAuthenticatedLinks] Redirecting to login: [#{error.class}] "\
-        "Could not determine current shop domain")
+      ShopifyApp::Logger.warn("[ShopifyApp::EnsureAuthenticatedLinks] Redirecting to login: [#{error.class}]"\
```

Comment:
It's only this way because this was an existing log entry. I think including the class/module name everytime is a little too noisy - perhaps we only include it if there's some similar code paths and we need to differentiate them?